Gemfile.lock

```diff
@@ -1,57 +1,57 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actionpack (5.2.1)
-      actionview (= 5.2.1)
-      activesupport (= 5.2.1)
+    actionpack (5.2.2.1)
+      actionview (= 5.2.2.1)
+      activesupport (= 5.2.2.1)
       rack (~> 2.0)
       rack-test (>= 0.6.3)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.0.2)
-    actionview (5.2.1)
-      activesupport (= 5.2.1)
+    actionview (5.2.2.1)
+      activesupport (= 5.2.2.1)
       builder (~> 3.1)
       erubi (~> 1.4)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.0.3)
-    activemodel (5.2.1)
-      activesupport (= 5.2.1)
-    activerecord (5.2.1)
-      activemodel (= 5.2.1)
-      activesupport (= 5.2.1)
+    activemodel (5.2.2.1)
+      activesupport (= 5.2.2.1)
+    activerecord (5.2.2.1)
+      activemodel (= 5.2.2.1)
+      activesupport (= 5.2.2.1)
       arel (>= 9.0)
-    activesupport (5.2.1)
+    activesupport (5.2.2.1)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
     arel (9.0.0)
     builder (3.2.3)
-    concurrent-ruby (1.1.1)
+    concurrent-ruby (1.1.5)
     crass (1.0.4)
     double_entry (1.0.1)
       activerecord (>= 3.2.0)
       activesupport (>= 3.2.0)
       money (>= 6.0.0)
       railties (>= 3.2.0)
-    erubi (1.7.1)
-    i18n (1.1.1)
+    erubi (1.8.0)
+    i18n (1.6.0)
       concurrent-ruby (~> 1.0)
-    json (2.1.0)
-    jwt (1.5.6)
-    jwt_signed_request (2.4.0)
-      jwt (~> 1.5.0)
+    json (2.2.0)
+    jwt (2.1.0)
+    jwt_signed_request (2.5.1)
+      jwt (>= 1.5.0, < 2.2.0)
       rack
     loofah (2.2.3)
       crass (~> 1.0.2)
       nokogiri (>= 1.5.9)
-    method_source (0.9.1)
-    mini_portile2 (2.3.0)
+    method_source (0.9.2)
+    mini_portile2 (2.4.0)
     minitest (5.11.3)
-    money (6.13.1)
+    money (6.13.2)
```

Comment:
### [money](https://rubymoney.github.io/money)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 6.13.1 → 6.13.2

[_[change-log](https://github.com/RubyMoney/money/blob/master/CHANGELOG.md), [source-code](https://github.com/RubyMoney/money/)_]





<details>
<summary>Commits</summary>

A change of **10** commits. See the full changes on [the compare page](https://github.com/RubyMoney/money/compare/v6.13.1...v6.13.2).

These are the individual commits:
- (5626e4c) [Disallow initializing Money with non-finite amounts](https://github.com/RubyMoney/money/commit/5626e4c12a2a39ffb4b6a32149d222e1265c418a)
- (b82492c) [Convert the fractional value of a Money object to BigDecimal…](https://github.com/RubyMoney/money/commit/b82492c8e27363e40a3519c85eb60b8206f5017e)
- (8f5ccc1) [Offer replacements for currency position deprecations (#833)](https://github.com/RubyMoney/money/commit/8f5ccc1d5238ee442c4e57aca89a124f68ecf6f7)
- (e1431f8) [Merge pull request #832 from antstorm/disallow-non-finite-am…](https://github.com/RubyMoney/money/commit/e1431f8fbab6400ff155520e7eac011a15825190)
- (9504501) [Add documentation for rounding (#834)](https://github.com/RubyMoney/money/commit/95045018d262a6751887a2d75e4775948e465881)
- (d904e71) [Fix Peruvian Sol symbol (#838)](https://github.com/RubyMoney/money/commit/d904e7160c9bf7f0b53bfff7c46df15f711f9c0d)
- (da1559d) [Fix bundler and i18n version dependencies (#842)](https://github.com/RubyMoney/money/commit/da1559d70549d756ae6c388728ca849d487038a5)
- (0ac7cf4) [Allow for :symbol lookup in LocaleBackened::I18n (#843)](https://github.com/RubyMoney/money/commit/0ac7cf4e250a17e0f4a8526ae6e41ae80b6d1087)
- (6da5812) [Prevent Divide By Zero in Money#allocate (#841)](https://github.com/RubyMoney/money/commit/6da58123e61145bee8e9478461c9505887aeb050)
- (ec5492f) [Update version to 6.13.2](https://github.com/RubyMoney/money/commit/ec5492f220e2c4d9fb25bb6be6cb9f33abaa29c6)


</details>